dask_expr/expr.py

```diff
@@ -871,6 +875,29 @@ class Replace(Blockwise):
     operation = M.replace
 
 
+class _Align(Blockwise):
+    _parameters = ["frame", "other", "join", "fill_value"]
+    _defaults = {"join": "outer", "fill_value": None}
+    _keyword_only = ["join", "fill_value"]
+    operation = M.align
```

Comment:
Is this truly blockwise though?  What if one index values is on one partition in one dataframe, but in another partition in the other dataframe?

Reply:
I forgot adding a Repartition call before this class. The whole operation isn't blockwise, but this class is